### ISSUE TITLE
Add steps to user registration page

### ISSUE THREAD

**Issue by andrelas1** [bitfindr/bitfindr#52]
1 - First name, last name, email and password
2 - Username
3 - Register confirmation

**Comment by darzouras** [comment]
For clarification, would 'register confirmation' be satisfied by moving to a confirmation page after submission?

**Comment by andrelas1** [comment]
Yes, or we could use a modal or some sort of alert to give explicit feedback to the user.

**Comment by yannbf** [comment]
Hey @darzouras, thanks for your interest in the project! 

Yes, this would be something like a multistep registration, but being careful not to add too many steps to not bother the user experience. At the end, there could be a way to give feedback to the user, or just jump straight to home, like most apps do.

Feel free to get in our [slack channel](https://join.slack.com/t/bitfindr/shared_invite/enQtMjYwMTAzMzgwMDAzLWZmNjY4NTFhOTgwNzRmNDI0NDAyY2M1OWRjNDI3NjUyM2RkNWY0MmVjMzNhMWE0ZmJiZGE0ZmJkOTBhOTE2Mzk) to further discuss this with @andrelas1, which started working on this, I believe.

**Comment by andrelas1** [comment]
Yes @darzouras, I did started working on this, but please feel free to submit and discuss anything and if you are interested, our slack channel is where we discuss ideas and features for the Bitfindr project.

Thanks!
